### DISCUSSION TITLE
failed to render shortcode "img": failed to process shortcode:

### DISCUSSION THREAD

**Discussion by NoneelPrasad** [thuliteio/doks#450] [General]
Hi, I've been following the documentation for adding images to pages however I get  failed to render shortcode "img": failed to process shortcode: ........at <$image.Resize>: nil pointer evaluating resource.Resource.Resize

Has anyone recently faced this issue and how do you go about fixing it.
Very new to doks.

**Comment by h-enk** [comment]
@NoneelPrasad, do you have a repo you could share?

**Reply by NoneelPrasad** [reply]
Note I am very new to this and currently just leaning to use it.
Things that I have done so far:
 - used Doks child theme
 - changed menu and few file content
 - deployed to GitHub pages 
 - tried adding images using "{{< img src="rectangle.png" alt="Rectangle" caption="<em>Rectangle</em>" class="border-0" >}}" and couldn't get it to show
 
https://github.com/NoneelPrasad/covidtrackerdocument

**Reply by h-enk** [reply]
Use the shortcodes with images put in a page bundle, so e.g.:

![Snag_1f024841](https://user-images.githubusercontent.com/3902872/132459171-1add7fad-a5fb-4c6f-8598-a06bde413f59.png)

```md
### Country Stats

{{< img src="stats.jpg" alt="Rectangle" caption="<em>Rectangle</em>" class="border-0" >}}

```

![Snag_1f00c2ca](https://user-images.githubusercontent.com/3902872/132458935-a8d41f57-a17d-4d2d-aabf-720f0e02203c.png)

**Reply by NoneelPrasad** [reply]
@h-enk many thanks.
I completely overlooked the page bundle part.
Everything works as the tutorial now.
